dts/bindings/wifi/wifi-tx-power-2g.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  This is a representation of the Wi-Fi transmit power ceilings
+  i.e., maximum transmit power allowed for 2.4GHz band. Based on the
+  RF performance of the device, the transmit power can be limited
+  for different data rates and sub-bands to get the best performance.
+  Resolution is 1 dBm
+
+properties:
+  max-pwr-2g-dsss:
```

Comment:
These properties could be appended `wifi-` primarily to ensure downstream users are forced to update the properties in their dts and align with the new resolution. Secondarily it namespaces the properties nicely to they match the binding they are from :)